Examples/Sample_xDscWebServiceRegistrationWithSecurityBestPractices.ps1

```diff
@@ -53,7 +53,7 @@ configuration Sample_xDscWebServiceRegistrationWithEnhancedSecurity
             DependsOn               = "[WindowsFeature]DSCServiceFeature" 
             RegistrationKeyPath     = "$env:PROGRAMFILES\WindowsPowerShell\DscService"   
             AcceptSelfSignedCertificates = $true
-            UseUpToDateSecuritySettings = $true
+            UseSecurityBestPractices = $true
```

Comment:
If a previous configuration exists, users could restore to that one. Otherwise, there's no way for us to figure out the settings before applying this configuration and you are correct about there will not be a way to back out. This is the issue for all DSC configurations and reason to introduce inventory work.